python/ngen_cal/setup.cfg

```diff
@@ -36,6 +36,7 @@ install_requires =
     hydrotools.metrics
     hydrotools.nwis_client
     pyarrow
+    pyswarms
```

Comment:
Yeah? I am inclined to remove the root repo `requirements.txt` in favor of a ns package level `requirements.txt` instead. In either case, we should use `pip freeze` to generate `requirements.txt` instead of just listing out all the deps.

Reply:
Lets open this as a TODO in an issue and cycle back to this.